src/store/reducers/appReducer.js

```diff
@@ -46,6 +63,24 @@ export const epics = createEpicScenario({
         .catch(handleError(fromAction))
   },
 
+  /** Get solution settings */
+  getSolutionSettings: {
+    type: 'APP_SOLUTION_GET_SETTINGS',
```

Comment:
When getSolutionSettings is renamed to match the other fetchables... this will need to be renamed with "FETCH" as well.

Reply:
done